pt/endpoints/data/model.py

```diff
@@ -0,0 +1,121 @@
+import uuid
+from config import db
+
+# pylint: disable=W0611
+from ..address.model import History # NOQA
+
+# pylint: enable=W0611
+
+
+class Data(db.Model):
+    uuid = db.Column(db.String(40), primary_key=True, unique=True, nullable=False)
+    field = db.Column(db.String(120), unique=False, nullable=False)
+    updated_at = db.Column(db.DateTime, unique=False, nullable=False)
+    address = db.Column(db.String(120))
+    # ForeignKey to History
+    history_id = db.Column(db.String(80), db.ForeignKey('history.uuid'), nullable=False)
+    history = db.relationship('History')
+    data_type = db.Column(db.String(50))
+    __mapper_args__ = {'polymorphic_identity': 'Data', 'polymorphic_on': data_type}
+
+    def __init__(self, field, updated_at, history_id, address):
+        self.uuid = str(uuid.uuid4())
+        self.field = field
+        self.updated_at = updated_at
+        self.history_id = history_id
+        self.address = address
+
+    def add(self):
+        db.session.add(self)
+        db.session.commit()
+
+    # pylint: disable=R0201
+    def update(self):
+        db.session.commit()
+    # pylint: enable=R0201
+
+    def delete(self):
+        db.session.delete(self)
+        db.session.commit()
+
+    def __repr__(self):
+        return self.uuid
+
+
+class Homepage(Data):
+    __tablename__ = 'homepage'
```

Comment:
Suggest not naming a table 'homepage'

It is not readable for recognizing what is inside.

Reply:
Rename homepage -> demand